lib/awful/screen.lua.in

```diff
@@ -104,6 +104,12 @@ function screen.padding(_screen, padding)
     return data.padding[_screen]
 end
 
+--- Get the focused screen.
+-- This can be replaced in a user's config.
+function screen.focused()
+    return capi.mouse.screen
```

Comment:
I have a key binding that moves the mouse pointer to the other screen. This allows me to e.g. switch tags on "the other screen", even if there is no client on that screen (input focus stays on some other screen).